apps/web/preset.config.mjs

```diff
@@ -20,7 +20,7 @@ export const blockSuiteFeatureFlags = {
 export const buildFlags = {
   enableImagePreviewModal: process.env.ENABLE_IMAGE_PREVIEW_MODAL
     ? process.env.ENABLE_IMAGE_PREVIEW_MODAL === 'true'
-    : true,
+    : false,
```

Comment:
No, you should enable it by default, but disable it in the CI

Reply:
Do you mean that we just add a new environment variable in GitHub settings?

Reply:
Oh you mean update the workflow files directly?